searx/static/themes/simple/src/less/code.less

```diff
@@ -1,9 +1,51 @@
-.code-highlight pre {
-  overflow: auto;
-  background-color: inherit;
-  color: inherit;
-  border: inherit;
+@import "../generated/pygments.less";
+
+.codelines {
+  margin: @results-margin 0 0 0;
+  padding: @result-padding 0 0 0;
 }
 
-// stylelint-disable no-invalid-position-at-import-rule
-@import "../generated/pygments.less";
+.code-highlight-sxng() {
+  .code-highlight {
+    pre {
+      overflow: auto;
+      margin: 0;
+      padding: 0 0 0.75rem 0;
+    }
+
+    .linenos {
+      user-select: none;
+      cursor: default;
+
+      &::selection {
+        background: transparent; /* WebKit/Blink Browsers */
+      }
+
+      &::-moz-selection {
+        background: transparent; /* Gecko Browsers */
+      }
+      margin-right: 8px;
+      text-align: right;
+    }
+
+    span.linenos {
+      color: #64708d;
```

Comment:
Not sure if it is a good idea to custom the line.nummbers color here / but in the "default" style and in "lightbulb" style the color was to dark .. a value of `#64708d;` fits to both styles .. but when we decide to use other pygments style we have to modify or remove the value here.

Reply:
I agree with you on that.
The line numbers are a bit too highlighted for my taste, it'd be cooler to use a gray color for it.

Nevertheless I don't think that's worth doing any extra patches only to change the line number color, for what its worth most people probably won't notice it anyways :)